Libraries/WebSocket/RCTSRWebSocket.m

```diff
@@ -530,7 +530,14 @@ - (void)_initializeStreams
 - (void)_connect
 {
   if (!_scheduledRunloops.count) {
-    [self scheduleInRunLoop:[NSRunLoop RCTSR_networkRunLoop] forMode:NSDefaultRunLoopMode];
+    // [TODO(macOS ISS#2323203): `scheduleInRunLoop:forMode:` takes in a non-null run loop parameter so let's be safe and verify that
```

Comment:
Do we have a better bug number or can we create one in git then this generic ISS number?

Reply:
I looked and one didn't exist. I figure since we need to go through and update all of these anyway, making it one that we can easily track/update all at once might be easier than making a new one and not tracking it well/forgetting about it